generators/client/templates/angularjs/src/main/webapp/app/blocks/handlers/_translation.handler.js

```diff
@@ -44,6 +47,24 @@
             });
         }
 
+      <%_ if (enableRTLSupport) { _%>
+        // Update language and direction in index.html, e.g. <html dir="ltr" or <html dir="rtl"
+        function updatePageDirection() {
+          var currentLang = $translate.proposedLanguage() || $translate.use();
+          angular.element('html').attr('lang', currentLang);
+          angular.element('html').attr('dir', isRTL(currentLang) ? 'rtl' : 'ltr');
+        }
+
+        // Returns true if passed language key is a Right-to-Left language key
+        function isRTL(langKey) {
```

Comment:
I would like to get this from the existing language filter. See my comment below